src/lib389/lib389/backend.py

```diff
@@ -510,6 +510,82 @@ def _lint_mappingtree(self):
             result['items'] = [bename, ]
             yield result
 
+
+    def _lint_backend_implementation(self):
+        backend_impl = self._instance.get_db_lib()
+        if backend_impl == 'bdb':
+            result = DSBLE0006
+            result['items'] = [self.lint_uid()]
+            yield result
+
+
+    def _lint_backend_implementation_mismatch(self):
```

Comment:
I wonder if this should be tested.
FYI: If they are not already existing, the dbfiles are created when the server is started.
So such a warning will be transient  (and seen when performing a manual migration or having explicitly removed the db files)

Reply:
Same here; it's more of a clean-up, low-severity health check, so the administrator is aware that some things still need to be done for a clean migration finish.

But I don't have a strong opinion about this one. If you think it's really unnecessary, I'm okay with removing it.

Reply:
The more I think about it, the more I think that the interesting warning is not if the db files are missing 
 because simply restarting the server will fix recreate them but rather if the db file for the both implementations are present (meaning that both database `dsctl instance dblib cleanup` has not yet been run)
  and it is still a minor severity (Useless data not removed)